server/index.ts

```diff
@@ -1,8 +1,16 @@
-import app from "./server";
+import GetNodeServer from "./server";
 import pino from "pino";
+import { loadConfigFromEnv } from "./Config";
+import BlaiseApiClient from "blaise-api-node-client";
+import { Auth } from "blaise-login-react/blaise-login-react-server";
 
 const port: string = process.env.PORT || "5002";
 const logger = pino();
-app.listen(port);
+
+const config = loadConfigFromEnv();
```

Comment:
I'm testing this line on my local dev which should not show can't find env variable for the entries in .env flie. I will update here once i resolved the issue.